plugp100/discovery/tapo_discovery.py

```diff
@@ -0,0 +1,208 @@
+import ipaddress
+import logging
+import struct
+import sys
+import zlib
+import json
+import base64
+import socket
+import io
+import select
+import base64
+import os
+import time
+
+from Crypto.Cipher import PKCS1_v1_5
```

Comment:
I've removed pycryptodome with cryptography, cause has issue related to pypi wheels for different CPU architecture. Can you use cryptography?